components/profile-controller/controllers/profile_controller.go

```diff
@@ -105,6 +101,7 @@ type ProfileReconciler struct {
 func (r *ProfileReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 	ctx := context.Background()
 	logger := r.Log.WithValues("profile", request.NamespacedName)
+	defaultKubeflowNamespaceLabels = r.DefaultNamespaceLabels
```

Comment:
nit: it's usually not a good idea to use global variables

shall we pass this argument explicitly?

Reply:
Good point! I have changed to use argument explicitly.

Reply:
Did you forget to push some commits? or did I miss anything?
Looks like the code is still using the global variable `defaultKubeflowNamespaceLabels`

Reply:
Good point! I should use local variable instead. Updated and added README.